DistroLauncher/extended_cli_parser.cpp

```diff
@@ -54,6 +54,11 @@ namespace Oobe::internal
 
     Opts parse(const std::vector<std::wstring_view>& arguments)
     {
+        // launcher.exe --hide-console - Windows Shell GUI invocation as declared in the appxmanifest. Hides the
```

Comment:
mismatch: the option is named `--from-start-menu-or-store`.

TBH, I prefer `hide-console` which is what the option will do instead of basing on a policy based on where it starts from (and which can evolve)

Reply:
That was the original idea, I changed my mind due the fact that later we may want to do other things besides hidding the console, but I guess you're right. I'm reverting to `--hide-console`.